candle-core/examples/safetensors.rs

```diff
@@ -0,0 +1,9 @@
+use candle_core::quantized::{ggml_file::qtensor_from_ggml, GgmlDType};
+use candle_core::Device;
+
+fn main() {
```

Comment:
keeping this present on this branch to repro the issue from https://github.com/huggingface/candle/issues/2040

Reply:
@tomsanbear As I saw, most of the `GgmlType` have larger alignments than `u8`. How do we make sure we use `qtensor_from_ggml` and get the correct data value?